Manager/ViewModel/Demos/DemoListViewModel.cs

```diff
@@ -1631,6 +1631,7 @@ private void HandleMainWindowLoadedMessage(MainWindowLoadedMessage msg)
 				HasRing = true;
 				NotificationMessage = "Initializing cache...";
 				await _cacheService.InitDemoBasicDataList();
+				await _cacheService.InitDemoCache();
```

Comment:
This should not be here since the demos list view already load demos data and doesn't need to store it in memory cache because when the user change the folder, refresh the list... the whole folder have to be reloaded to detect added / removed demos.
It's increasing RAM usage and should be done only for the stats part.
An other solution would be to do it here and use the same cache in all parts of the app but it required to re-write some logics (LoadDemosHeader() for example).